util/parsenum.h

```diff
@@ -141,7 +154,7 @@ _Pragma("clang diagnostic pop")
 	)
 
 /* Magic to select which version of PARSENUM_BASE to use. */
-#define PARSENUM_BASE(...)	PARSENUM_BASE_(PARSENUM_BASE_COUNT(__VA_ARGS__))(__VA_ARGS__)
+#define PARSENUM_BASE(...)	PARSENUM_BASE_(PARSENUM_BASE_COUNT(__VA_ARGS__))(__VA_ARGS__, "PARSENUM_BASE")
```

Comment:
The `_define_str` is added here.  When `PARSENUM` is switched to call `PARSNEUM_EX{3,5}`, those lines will set it to the relevant string.